### ISSUE TITLE
var readers = [ 'mendeley', 'connotea', 'citeulike' ];

### ISSUE THREAD

**Issue by photomedia** [eprintsug/altmetric#1]
Connotea discontinued service on March 12, 2013.

in 99_altmetric.js we have:
var readers = [ 'mendeley', 'connotea', 'citeulike' ];

Should remove Connotea from the list, because it will be 0.

**Comment by photomedia** [comment]
About this, it may be better to modify this line so that none of the 0 result readers show up:

in 99_altmetrics.js we have:
line 73  if (value != null)

Change to the following to remove "0" results in the readers:
line 73 if(( value != null ) && (value != 0))

**Comment by jesusbagpuss** [comment]
@photomedia - do you want to try installing the latest package on a dev repo - in case I've made any mistakes (seems to work OK on 3.3.10)?
It will now use DOIs or ISBNs to display data. An example ISBN that actually has some data attached is: 978-3-319-25557-6 - in case you need it to test properly.

**Comment by photomedia** [comment]
Thank you for the updates! Sorry for the delay, it's a very busy time of year.  The changes look OK to me, but I'm unsure why you added an extra screen and what permissions are required for accessing the screen?  I didn't try this update yet on a repository, but when I do, I will report how it goes.

**Comment by jesusbagpuss** [comment]
Hi Tomsaz,
The Altmetric API can handle DOIs, ISBNs, PMIDs, and other identifiers.
This package will allow any of these to be used - but as they may exist in other fields, I added a config function to allow someone to alter this at the repo level.
The screen I added is for repo admins - and makes changing this function easier (and also adding the API key if you have one).

I've tested this in 3.3.10, and made it live in production. I didn't want to push it to the Bazaar until I've got more evidence that it's good!
